detox/android/detox/src/coreNative/java/com/wix/detox/actions/DetoxViewActions.kt

```diff
@@ -32,7 +29,4 @@ public object DetoxViewActions {
 
     public fun scrollRightBy(amountInDp: Double, startOffsetPercentX: Float? = null, startOffsetPercentY: Float? = null): ViewAction =
         actionWithAssertions(DetoxScrollAction(MOTION_DIR_RIGHT, amountInDp, startOffsetPercentX, startOffsetPercentY))
-
-    public fun takeScreenshot(): ViewActionWithResult<ScreenshotResult?> =
```

Comment:
What happened, @d4vidi? Why? I am curious.

Reply:
Oh I actually described that briefly in the issue's description